### PR TITLE
Creating footer

### PR DESCRIPTION
**Pull request by pathto493**
Made changes to earlier Index Page (earlier version was done wrongly as explained by Prima
- did a redo on difficulty bar
- did a redo on the video card
- code is much lesser in index page because of the use of _video and _difficultybar 
- added a padding to seperate the navigation bar and the video card
Created Footer
- made the icon size smaller
- could not decrease the footer size, need ask for help tomorrow

### CODE REVIEW

File: app/views/videos/index.html.erb [hunk 1]
```diff
@@ -1,92 +1,15 @@
...
-                      Sorry, your browser doesn't support embedded videos. <!-- Alternate text if the video cannot be not shown -->
-                    </video>
-                    </div>
-                    <h4><%= link_to video.title, video_path(video.id) %></h4>
-                    <h6>by who |  <%= video.duration %>mins | difficulty level: <%= video.difficulty %></h6>
-                  </div>
-              <% end %>
-                <%# <div class="card-display">
-                  <h5>
-                    <img src="https://images.unsplash.com/photo-1575052814086-f385e2e2ad1b?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&auto=format&fit=crop&w=1350&q=80" alt="yoga">
```
Comment (nielsantosa) [inline app/views/videos/index.html.erb:55]: Remove this repetition of codes

File: app/views/videos/backup.html.erb [hunk 2]
```diff
@@ -0,0 +1,58 @@
+<div class="main-container">
+  <div class="container">
```
Comment (nielsantosa) [inline app/views/videos/backup.html.erb]: What is this backup for?

File: app/views/videos/index.html.erb (hunk 1 above)
Comment (pathto493) [reply app/views/videos/index.html.erb:55]: okay

Comment (pathto493) [reply app/views/videos/index.html.erb:55]: i think that might not be the latest version

File: app/views/videos/backup.html.erb (hunk 2 above)
Comment (pathto493) [reply app/views/videos/backup.html.erb]: oh i think can take away, prima guided me on a new way of doing the index page. He created a backup to keep what i done before

File: app/views/videos/index.html.erb (hunk 1 above)
Comment (pathto493) [reply app/views/videos/index.html.erb:55]: image

**Comment by pathto493** [comment]
I deleted the backup

File: app/views/videos/backup.html.erb (hunk 2 above)
Comment (nielsantosa) [reply app/views/videos/backup.html.erb]: ok, then can keep it.

File: app/views/videos/index.html.erb (hunk 1 above)
Comment (nielsantosa) [reply app/views/videos/index.html.erb:55]: oh ya, my fault, red color means removed already. looks ok.

**Review by nielsantosa — APPROVED** [review]

**Pull request merged**